Submodels/MTSOL_call.py

```
@@ -685,60 +663,55 @@ def HandleNonConvergence(self,
 
             # Generate solver outputs
-            self.GenerateSolverOutput()
+            self.GenerateSolverOutput(output_type=OutputType.FORCES_ONLY)
 
             # Rename file to indicate the iteration number, and avoid overwriting the same file. 
             # Also move the file to the output folder
             # Waits for the file to exist before copying.
             init_time = time.time()
-            timer = 0
-            while not event_handler.is_file_processed() and timer < 10:
+            while not event_handler.is_file_processed() and (current_time - init_time) <= 10:
```

Ruff (0.8.2)

Undefined name `current_time`

(F821)